w3d_translator/unity.py

```diff
@@ -1,7 +1,50 @@
-# Path to Unity executable
+import subprocess
+import shutil
+import os
+
+from errors import UnityException
+
+
 UNITY_VERSION = "2020.3.26f1"
-UNITY_PATH = "C:\\Program Files\\Unity\\Hub\\Editor\\2020.3.26f1\\Editor\\Unity.exe"
+UNITY_PATH = "C:\\Program Files\\Unity\\Hub\\Editor\\2020.3.26f1\\Editor\\Unity.exe"  # noqa (ignore lint)
+EMPTY_SCENE = "../EmptyScene.unity"
+
 
-# TODO Create a Unity project at the given folder
+# Create Unity project
 def create_project(folder):
-    print('create_project' + folder)
\ No newline at end of file
+    try:
+        subprocess.run(
+            f'{UNITY_PATH} -batchmode -createProject "{folder}" -quit'
+        )
+    except Exception:
+        raise UnityException(
+            "Error: Failed to launch Unity.\n"
+            + "This script only runs with "
+            + f"Unity version {UNITY_VERSION}.\n"
+            + f"The executable should be found at {UNITY_PATH}"
+        )
+
+
+# Copy project into [unity_dir]/Assets/originals
+def copy_files(project_dir, unity_dir):
+    destination = os.path.join(unity_dir, "Assets", "originals")
+    try:
+        shutil.copytree(project_dir, destination)
+    except Exception as e:
+        raise UnityException(
+            f"Error: Failed to copy files from {project_dir} "
+            + f"to {destination}.\n"
+            + f"{e}"
+        )
+
+
+# Create Scenes folder and add empty scene
+def add_empty_scene(unity_dir):
+    try:
+        destination = os.path.join(unity_dir, "Assets", "Scenes")
+        os.mkdir(destination)
+        shutil.copy2(EMPTY_SCENE, os.path.join(destination, "Scene.unity"))
```

Comment:
The new hotness with making directories and copying things and whatnot is [pathlib](https://docs.python.org/3/library/pathlib.html). It's nice and doesn't require you to use 3 different APIs

Reply:
Will have to check it out!

Reply:
Added `pathlib`!